force-app/main/default/classes/AccountTriggerHandler.cls

```diff
@@ -0,0 +1,42 @@
+public with sharing class AccountTriggerHandler {
+    
+    public static void beforeInsert(List<Account> beforeInsertedAccounts) {
+        for(Account acc : beforeInsertedAccounts){
+        //Question 1
+            if(String.isBlank(acc.Type)){
+                acc.Type = 'Prospect';
+            }
+        //Question 2
+            if (!String.isBlank(acc.ShippingStreet) || 
+                !String.isBlank(acc.ShippingCity) || 
+                !String.isBlank(acc.ShippingState) || 
+                !String.isBlank(acc.ShippingPostalCode) || 
+                !String.isBlank(acc.ShippingCountry)) {
+                        acc.BillingStreet = acc.ShippingStreet;
+                        acc.BillingCity = acc.ShippingCity;
+                        acc.BillingState = acc.ShippingState;
+                        acc.BillingPostalCode = acc.ShippingPostalCode;
+                        acc.BillingCountry = acc.ShippingCountry;
+                    }
+            
+
+        //Question 3
+            if(acc.Phone != null && acc.Fax != null && acc.Website != null){
```

Comment:
Great work on these! It might also be helpful as a future enhancement to separate each piece of logic into its own method